tests/test.py

```diff
@@ -1,10 +1,10 @@
-# coding=utf-8
+# -*- coding: utf-8 -*-
 """Unit test for case-conversion
 """
 
 import case_conversion
 from unittest import TestCase
-from nose_parameterized import parameterized
+from parameterized import parameterized
```

Comment:
Nice. I'm going to add a `requirements.txt` file and a MakeFile I like to use for Python projects now, so testing will be easier. Just run `make test`. This will make it easier to add CI as well. I'll do this after we merge your PR

Reply:
You got the deprecation warning for `nose_parameterized`?
Also do you want to stick to requriements.txt or use the new pipfile format?
In the former case it may make sense to also add a `requirements-dev.txt`.

Reply:
Not aware of the new pipfile format. I'm down for that.

Reply:
One of Kenneth Reitz' most recent projects is `pipenv`. It provides package and virtual environment management in a unified interface and also features pipfiles for deterministic builds . It is super cool:
https://github.com/pypa/pipenv

I will give the question on which way to declare out dependencies some more thought.